docs/configuration.md

```diff
@@ -139,7 +139,7 @@ mining:
 ```
 in the yaml.epoch configuration file.
 
-Your mining setup needs to meet your hardware capacity. Therefore, you need to make a choice in how you want to configure your miner. You can read the documentation on setting up CUDA mining, or you can use all but one of the cores on your computer to mine (keep one core available for transaction gossiping, synchronising, etc).
+Your mining setup needs to meet your hardware capacity. Therefore, you need to make a choice in how you want to configure your miner. You can read the documentation on setting up CUDA mining, or you can use all but one of the cores on your computer to mine (keep one core available for transaction gossiping, synchronising, etc). [more details: about epoch_config_schema](https://github.com/aeternity/epoch/blob/master/apps/aeutils/priv/epoch_config_schema.json)
```

Comment:
Could you please make this a local link to the branch (opposed to always pointing to master):
```
[more details: about epoch_config_schema](../apps/aeutils/priv/epoch_config_schema.json)
```